docs/development/deploying-at-scale/index.md

```diff
@@ -1,3 +1,41 @@
 # Deploying at scale
 
 REANA can be easily deployed on large Kubernetes clusters using Helm. Useful for production instances.
+
+## Pre-requisites
+
+REANA needs a shared file system to host all analyses workspaces when running in a multinode deployment setup. Therefore, you should create an [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource) pointing to your storage backend. The `StorageClass` should meet the following requirements:
+
+- Be named `<helm-release-prefix>-shared-volume-storage-class`
```

Comment:
- be named ... ;
- be created ... .

i.e. usually the items start in lowercase and end by `;` except for the last item that ends by `.`.